### PR TITLE
getauxval isn't defined on uclibc, so disable dynamic feature detection

### PR DESCRIPTION
**Pull request by skrap**
This is my attempt at the suggested alternative approach @briansmith suggested in https://github.com/briansmith/ring/pull/1800

### CODE REVIEW

File: src/cpu/arm.rs [hunk 1]
```diff
@@ -19,7 +19,8 @@
 
 #[cfg(all(
```
Comment (briansmith) [inline src/cpu/arm.rs:20]: Please add a comment above this along the following lines (with any typos fixed):
> uclibc: When linked statically, uclibc doesn't provide `getauxval`. When linked dynamically, recent versions do provide it, but we want to support older versions too. Assume that if uclibc is being used, this is an embedded target where the user cares a lot about minimizing code size and also that they know in advance exactly what target features are supported, so rely only on static feature detection.

Comment (skrap) [reply src/cpu/arm.rs:28]: Done!

**Review by briansmith — APPROVED** [review]

**Comment by skrap** [comment]
Drat I forgot to run rustfmt after adding the comment.  Fixed.  Sorry!

**Comment by skrap** [comment]
Thank you!

**Pull request merged**